packages/util-linux/util-linux.spec

```diff
@@ -299,7 +304,6 @@ done
 %dir %{_cross_includedir}/blkid
 %{_cross_includedir}/blkid/blkid.h
 %{_cross_pkgconfigdir}/blkid.pc
-%exclude %{_cross_libdir}/libblkid.la
```

Comment:
Do we know why these disappeared?

Reply:
I can't seem to find why these are coming into the build now, but its more of a curiosity, not a blocking issue.